google/cloud/aiplatform/models.py

```diff
@@ -1371,51 +1881,189 @@ def list(
                 credentials set in aiplatform.init.
 
         Returns:
-            List[models.Endpoint] - A list of Endpoint resource objects
+            List[models.PrivateEndpoint] - A list of PrivateEndpoint resource objects
         """
 
         return cls._list_with_local_order(
+            cls_filter=lambda ep: bool(
+                ep.network
+            ),  # Only private Endpoints have a network set
             filter=filter,
             order_by=order_by,
             project=project,
             location=location,
             credentials=credentials,
         )
 
-    def list_models(self) -> Sequence[gca_endpoint_v1.DeployedModel]:
-        """Returns a list of the models deployed to this Endpoint.
-
-        Returns:
-            deployed_models (Sequence[aiplatform.gapic.DeployedModel]):
-                A list of the models deployed in this Endpoint.
-        """
-        self._sync_gca_resource()
-        return self._gca_resource.deployed_models
+    def deploy(
+        self,
+        model: "Model",
+        deployed_model_display_name: Optional[str] = None,
+        traffic_percentage: Optional[int] = 100,
```

Comment:
If I understand correctly, only one Model can be deployed to a PrivateEndpoint. What's the purpose of exposing `traffic_percentage` and `traffic_split`?

Reply:
Fixed, removing these arguments and explicitly setting `traffic_percentage=100` and `traffic_split=None`